content/en/blog/_posts/2018-03-00-How-To-Integrate-Rollingupdate-Strategy.md

```diff
@@ -168,8 +168,6 @@ items:
 
    resourceVersion: "415852"
 
-   selfLink: /apis/mysql.orain.com/v1/namespaces/default/mysqlclusters/clustershard-c
```

Comment:
It sounds OK to me to leave these references in blog posts. That information was appropriate at the time, right?